### PR TITLE
[DOCS-7164] Alfresco Transform Service 2.1

### PR DESCRIPTION
**Pull request by Amy-Hyland**

### CODE REVIEW

**Review by anxumalo — CHANGES_REQUESTED** [review]
@Amy-Hyland  Sending early feedback for your consideration

File: transform-service/latest/support/index.md
[the remark this answers is not in the document]
Comment (Amy-Hyland) [reply transform-service/latest/support/index.md:1]: done

File: transform-service/latest/install/index.md [hunk 1]
```diff
@@ -269,7 +269,7 @@ from the left column that corresponds to the required Content Services version y
     ```
 
     > **Note:** Replace the version number `x.y.z` with the tag that matches the Content Services version you want to 
-    > deploy. For example, if you want Content Services 7.3.0, then select tag `5.3.0`.
+    > deploy. For example, if you want Content Services 7.3.0, then select tag `7.3.0`.
```
[the remark this answers is not in the document]
Comment (Amy-Hyland) [reply transform-service/latest/install/index.md]: I changed the 5.3.0 example back.

File: transform-service/latest/install/index.md
[the remark this answers is not in the document]
Comment (Amy-Hyland) [reply transform-service/latest/install/index.md:1]: They should be included now.

**Review by HeinRagas — APPROVED** [review]
LGTM

File: transform-service/latest/install/index.md [hunk 2]
```diff
@@ -269,7 +269,7 @@ from the left column that corresponds to the required Content Services version y
     ```
 
     > **Note:** Replace the version number `x.y.z` with the tag that matches the Content Services version you want to 
-    > deploy. For example, if you want Content Services 7.3.0, then select tag `7.3.0`.
+    > deploy. For example, if you want Content Services 7.3.0, then select tag `5.3.0`.
```
Comment (anxumalo) [inline transform-service/latest/install/index.md]: Let's change the example to ACS **7.4.0** - tag `6.0.0`

Comment (anxumalo) [reply transform-service/latest/install/index.md]: Updated in commit [782fa07](https://github.com/Alfresco/docs-alfresco/pull/1017/commits/782fa07d8413e0d9cbcd31bb19c6d5ee8df111bf)

**Pull request merged**